restaurants/views.py

```diff
@@ -1,3 +1,19 @@
-from django.shortcuts import render
+from django.http import JsonResponse
+from django.views import View
 
-# Create your views here.
+from restaurants.models import SubCategory
+
+class MainListView(View):
```

Comment:
Main <- 메인 페이지 기준으로 엔드포인트를 만드시면 안됩니다!

```suggestion
class SubCategoryListView(View):

```